packages/concerto-core/lib/serializer/jsonpopulator.js

```diff
@@ -94,13 +94,11 @@ class JSONPopulator {
      * Constructor.
      * @param {boolean} [acceptResourcesForRelationships] Permit resources in the
      * place of relationships, false by default.
-     * @param {boolean} [ergo] target ergo.
      * @param {number} [utcOffset] - UTC Offset for DateTime values.
      * @param {number} [strictQualifiedDateTimes] - Only allow fully-qualified date-times with offsets.
      */
-    constructor(acceptResourcesForRelationships, ergo, utcOffset, strictQualifiedDateTimes) {
+    constructor(acceptResourcesForRelationships, utcOffset, strictQualifiedDateTimes) {
```

Comment:
This is a breaking change for all upstream packages.

Reply:
I'm not sure about that, it's marked as a private package in JSDoc and not exported AFAIK

Reply:
If we're nervous, we can add a dummy param until the next major version

Reply:
Maybe I'm being too paranoid. I don't see any obvious uses of that in a code search?

Reply:
I'll go ahead and add a dummy parameter then. And note in the doc comment that it's a dummy parameter.